boom! testing git-workarea-1.0.1 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! d0d3d385f563686aff89fa79f6d46d3eb99843a60d83036eb8ade8c2ea90ec0d
boom! running `docker start -a d0d3d385f563686aff89fa79f6d46d3eb99843a60d83036eb8ade8c2ea90ec0d`
kablam!    Compiling git-workarea v1.0.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 3.98 secs
boom! running `docker rm -f d0d3d385f563686aff89fa79f6d46d3eb99843a60d83036eb8ade8c2ea90ec0d`
blam! d0d3d385f563686aff89fa79f6d46d3eb99843a60d83036eb8ade8c2ea90ec0d
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 73888e3d5875c65a60601a82d6caf21b94c9d9958ec4c3711786d2649a451722
boom! running `docker start -a 73888e3d5875c65a60601a82d6caf21b94c9d9958ec4c3711786d2649a451722`
kablam!    Compiling git-workarea v1.0.1 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 6.19 secs
boom! running `docker rm -f 73888e3d5875c65a60601a82d6caf21b94c9d9958ec4c3711786d2649a451722`
blam! 73888e3d5875c65a60601a82d6caf21b94c9d9958ec4c3711786d2649a451722
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! 4be1880fbf134a679f017fd553bf316ab208c21a529822758ff6df944a51f663
boom! running `docker start -a 4be1880fbf134a679f017fd553bf316ab208c21a529822758ff6df944a51f663`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/git_workarea-acc0334b83816a39
blam! 
blam! running 19 tests
blam! test test::test_mergeable ... FAILED
blam! test test::test_mergeable_conflict ... FAILED
blam! test test::test_gitdir ... FAILED
blam! test test::test_mergeable_already_merged ... FAILED
blam! test test::test_mergeable_no_common_history ... FAILED
blam! test test::test_reserve_ref ... FAILED
blam! test test::test_setup_merge ... FAILED
blam! test test::test_reserve_ref_no_exist ... FAILED
blam! test test::test_reserve_refs ... FAILED
blam! test test::test_setup_merge_submodule_conflict_resolution ... FAILED
blam! test test::test_setup_merge_conflict ... FAILED
blam! test test::test_setup_merge_submodule_not_merged ... FAILED
blam! test test::test_setup_merge_rewind ... FAILED
blam! test test::test_work_area_setup_no_exist ... FAILED
blam! test test::test_setup_merge_submodule_not_merged_conflict ... FAILED
blam! test test::test_setup_merge_submodule_not_present ... FAILED
blam! test test::test_work_area_setup ... FAILED
blam! test test::test_work_area_setup_submodule ... FAILED
blam! test test::test_work_area_setup_submodule_missing_links ... FAILED
blam! 
blam! failures:
blam! 
blam! ---- test::test_mergeable stdout ----
blam! 	thread 'test::test_mergeable' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- test::test_mergeable_conflict stdout ----
blam! 	thread 'test::test_mergeable_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_gitdir stdout ----
blam! 	thread 'test::test_gitdir' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_mergeable_already_merged stdout ----
blam! 	thread 'test::test_mergeable_already_merged' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_mergeable_no_common_history stdout ----
blam! 	thread 'test::test_mergeable_no_common_history' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_reserve_ref stdout ----
blam! 	thread 'test::test_reserve_ref' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge stdout ----
blam! 	thread 'test::test_setup_merge' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_reserve_ref_no_exist stdout ----
blam! 	thread 'test::test_reserve_ref_no_exist' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_reserve_refs stdout ----
blam! 	thread 'test::test_reserve_refs' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge_submodule_conflict_resolution stdout ----
blam! 	thread 'test::test_setup_merge_submodule_conflict_resolution' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_conflict stdout ----
blam! 	thread 'test::test_setup_merge_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge_submodule_not_merged stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_merged' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_rewind stdout ----
blam! 	thread 'test::test_setup_merge_rewind' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_work_area_setup_no_exist stdout ----
blam! 	thread 'test::test_work_area_setup_no_exist' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_setup_merge_submodule_not_merged_conflict stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_merged_conflict' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_setup_merge_submodule_not_present stdout ----
blam! 	thread 'test::test_setup_merge_submodule_not_present' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_work_area_setup stdout ----
blam! 	thread 'test::test_work_area_setup' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! ---- test::test_work_area_setup_submodule stdout ----
blam! 	thread 'test::test_work_area_setup_submodule' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:82
blam! 
blam! ---- test::test_work_area_setup_submodule_missing_links stdout ----
blam! 	thread 'test::test_work_area_setup_submodule_missing_links' panicked at 'origin clone failed: fatal: repository '/source/.git' does not exist
blam! ', src/test.rs:66
blam! 
blam! 
blam! failures:
blam!     test::test_gitdir
blam!     test::test_mergeable
blam!     test::test_mergeable_already_merged
blam!     test::test_mergeable_conflict
blam!     test::test_mergeable_no_common_history
blam!     test::test_reserve_ref
blam!     test::test_reserve_ref_no_exist
blam!     test::test_reserve_refs
blam!     test::test_setup_merge
blam!     test::test_setup_merge_conflict
blam!     test::test_setup_merge_rewind
blam!     test::test_setup_merge_submodule_conflict_resolution
blam!     test::test_setup_merge_submodule_not_merged
blam!     test::test_setup_merge_submodule_not_merged_conflict
blam!     test::test_setup_merge_submodule_not_present
blam!     test::test_work_area_setup
blam!     test::test_work_area_setup_no_exist
blam!     test::test_work_area_setup_submodule
blam!     test::test_work_area_setup_submodule_missing_links
blam! 
blam! test result: FAILED. 0 passed; 19 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f 4be1880fbf134a679f017fd553bf316ab208c21a529822758ff6df944a51f663`
blam! 4be1880fbf134a679f017fd553bf316ab208c21a529822758ff6df944a51f663
